src/Whoops/Resources/js/whoops.base.js

```diff
@@ -94,20 +104,34 @@ Zepto(function($) {
     return actionMsg;
   }
 
+  function scrollIntoView($node, $parent) {
+    var nodeOffset = $node.offset(),
+        nodeTop = nodeOffset.top,
```

Comment:
Regardless of personal preference, the surrounding code uses one `var` per variable style, so new code should do that, too.

- [ ] Use consistent var style